Gigs/Gigs/Application/GigController.swift

```diff
@@ -114,7 +114,7 @@ class GigController {
         
     }
     
-    func fetchAllGigs(completion: @escaping (Result<[Gig], NetworkError>) -> Void ){
+    func fetchAllGigs(completion: @escaping (Result<Bool, NetworkError>) -> Void ){
```

Comment:
You don't really need to do anything with the array of gigs in the call back, so the success result need just be a bool